### PR TITLE
Feature/APLAT-3769 horizontal Navbar

### PR DESCRIPTION
**Pull request by UshaKundapur**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "attainia-web-components",
-  "version": "2.0.0",
+  "version": "2.0.1",
```
Comment (mtomcal) [inline package.json]: Let's make this 2.1.0 since this is a new component.

File: src/components/index.js [hunk 2]
```diff
@@ -1 +1,7 @@
-export {default} from './button'
+import * as button from './button'
+import * as navbar from './navbar'
+
+export default {
+    button,
+    navbar
+}
```
Comment (mtomcal) [inline src/components/index.js]: We can do this instead:
```js
export * from './button'
export * from './navbar'
```

File: src/components/navbar/__tests__/navbarItem.test.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+import fakeConsole from '../../../__mocks__/fakeConsole'
+import NavbarItem from '../NavbarItem'
+
+const defaultProps = {
+    url: '#',
+    text: 'someText',
+    onClick: () => {}
+}
+const consoleSpy = fakeConsole()
+describe('Navbar', () => {
```
Comment (mtomcal) [inline src/components/navbar/__tests__/navbarItem.test.js]: ```describe('NavbarItem')```

File: src/components/navbar/_navbar.scss [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    text-align: center;
+    font-family: "Roboto";
+    color: #1c2a33;
+    background-color: #f4f7fb;
+    width:100px;
+    height: 22px;
+    line-height: 22px;
+    border: none;
+    &:hover {
+      background-color: #aedee9;
```
Comment (mtomcal) [inline src/components/navbar/_navbar.scss]: Should not drop in hex codes into component scss files. Why not use src/scss/_colors.scss instead?

Comment (mtomcal) [reply src/components/navbar/_navbar.scss]: Check out https://github.com/Attainia/attainia-web-components/blob/development/src/components/button/_button.scss for ideas on using color variables. You can also nest selectors for the menu styles like so with SCSS:
```scss
.menu {
    list-style: none;
    width: 100%;
    padding: 0;
    li {
      display: block;
      float:left;
      margin:-1px -1px 0 0;
   }
}
```

File: src/components/index.js (hunk 2 above)
Comment (UshaKundapur) [reply src/components/index.js]: When we do like this, it gives error `No named exports found in module './button'`. 
And When I change "default into named export like this :`export {button} from './Button'` in `../attainia-web-components/src/components/button/index.js` it gives eslint error `Prefer default export.` So thinking of changing back to as it was.

Comment (mtomcal) [reply src/components/index.js]: Alright thats fine!

File: src/components/navbar/_navbar.scss [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+          }
+        }
+      }
+    }
+    button {
+      display:block;
+      text-decoration:none;
+      font-size: 11px;
+      text-align: center;
+      font-family: "Roboto";
```
Comment (mtomcal) [inline src/components/navbar/_navbar.scss]: Shouldn't need font-family. Should be included in the base styles.

File: src/components/navbar/_navbar.scss [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+          &:last-of-type {
+            border-bottom: 1px solid $ui-3;
+          }
+        }
+      }
+    }
+    button {
+      display:block;
+      text-decoration:none;
+      font-size: 11px;
```
Comment (mtomcal) [inline src/components/navbar/_navbar.scss]: Should use from _typography.scss
```scss
@include typescale('11')
```

Comment (mtomcal) [reply src/components/navbar/_navbar.scss]: Fixed the comment to be font-size mixin. This helps us standardize our font scale.

File: src/components/navbar/_navbar.scss (hunk 5 above)
Comment (mtomcal) [reply src/components/navbar/_navbar.scss]: If not you can do ```@include font-family```

**Comment by mtomcal** [comment]
Once these two fixes are made. Ill approve. Just want to make sure the scss is maintainable in the long run by using all the benefits of our base styles.

**Pull request merged**